app.json

```diff
@@ -1,8 +1,8 @@
 {
   "expo": {
-    "name": "discovery-mobile-ui",
-    "slug": "discovery-mobile-ui",
-    "owner": "sync-for-science",
+    "name": "discovery-mobile-ui-karanl1",
+    "slug": "discovery-mobile-ui-karanl1",
+    "owner": "discovery-mobile-ui-karan",
```

Comment:
These config values should to be changed to a different organization.

Perhaps:  This file is converted to `app.js`, and these lines extract `EXPO_NAME`, `EXPO_NAME`, and `EXPO_NAME` from `process.env`.

(perhaps, using this github repo's `Settings > Secrets` to set them, for the expo build / CI.)